bin/data/rename_category

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../../config/environment"
+require_relative "../../config/cli"
+
+@cli = CLI.new([:name, :id]) do |opts|
+  opts.on("-n NAME", "--name=NAME", String, "New name for the category")
+  opts.on("-i ID", "--id=ID", String, "ID of the new category")
+end
+@cli.parse!(ARGV)
+@cli.options_status
+
+fail "Missing required option: --name" unless @cli.options.name
+fail "Missing required option: --id" unless @cli.options.id
+
+category = Category.find_by(id: @cli.options.id)
+fail "Category `#{@cli.options.id}` not found" unless category
+
+original_name = category[:name] # skip localizations
+original_handle = category.handleized_name
+category.update!(name: @cli.options.name)
+@cli.vputs("Updated category `#{category.id}` to `#{category[:name]}` from `#{original_name}` in DB")
+
+if category.handleized_name != original_handle
+  @cli.vputs("Renaming category file from `#{original_handle}.yml` to `#{category.handleized_name}.yml`")
+  @cli.move_file!("data/categories/#{original_handle}.yml", "data/categories/#{category.handleized_name}.yml")
+end
```

Comment:
Why does updating a category's name cause files to be moved? That would only be necessary if we're renaming a root category and therefore vertical. Will this not error for renaming non-root categories?

I assume most, if not all, renames will be on non-root categories

Reply:
I mean, we can fail on verticals too, but notionally a strict rename is allowed at that layer (this isn't a move, its really meant for spelling mistakes)